fashion-example-query/fashion/flow-query.yml

```diff
@@ -0,0 +1,16 @@
+!Flow
+with:
+  read_only: true  # better add this in the query time
+  logserver: $WITH_LOGSERVER
+  compress_hwm: 1024
+pods:
+  encode:
+    uses: encode/encode.yml
+    parallel: $PARALLEL
+  index:
```

Comment:
u will have to create 10 `parallel` indexers like this. In each of this, you have to put a `FilterQL` to ensure that it receives the documents with the `required` label